website/src/content/docs/internals/changelog.mdx

```diff
@@ -18,6 +18,26 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
 
 ### Analyzer
 
+### CLI
+
+### Configuration
+
+### Editors
+
+### Formatter
+
+### JavaScript APIs
+
+### Linter
+
+### Parser
+
+### VSCode
+
```

Comment:
We should not add the this. This disturbs changelog extraction for LSP releases.